_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,145 @@
+  such as `email_violations.py`
+
+  These notifiers can be:
+  1. `email_violations`
+  This emails all the violation data via SendGrid.  Multiple email recipients are
+  delimited by comma.
+
+  1. `slack_webhook`
+  This sends individual violations to a Slack channel via a Slack webhook.
+  Configure a webhook in your organization's Slack settings and set the `webhook_url`.
+
+  1. `gcs_violations`
+  This uploads all the violation data to a Cloud Storage bucket.
+
+* `data_format`: Either `csv` (default) or `JSON`.
+  Slack only support json type.
+
+The following example shows how to update a `.yaml` file to add email, Slack,
+and Cloud Storage notifier for Cloud SQL violations:
+
+```yaml
+notifier:
+    resources:
+        - resource: cloudsql_acl_violations
+          should_notify: true
+          notifiers:
+             - name: gcs_violations_pipeline
+               configuration:
+                 gcs_path: gs://path_to_foo_bucket
+             - name: email_violations_pipeline
+               configuration:
+                 sendgrid_api_key: foobar_key
+                 sender: forseti-notify@forsetisecurity.org
+                 recipient: foo@gmail.com,bar@gmail.com,baz@gmail.com
+             - name: slack_webhook_pipeline
+               configuration:
+                 webhook_url: https://hooks.slack.com/services/foobar
+```
+
+### Cloud SCC Findings
+
+Forseti violations can be output for integration with
+[Cloud SCC](https://cloud.google.com/security-command-center) on Google Cloud
+Platform (GCP).
+
+1. Open `forseti-security/configs/forseti_conf.yaml`.
+
+1. Navigate to the `notifier` > `violation` > `cloud_scc` section.
+
+```yaml
+notifier:
+    violation:
+      cloud_scc:
+        enabled: true
+        gcs_path: gs://inventoryscanner-henry.appspot.com
+```
+
+* `enabled`: Controls whether Cloud SCC findings should be sent.
+  `true` enables the notification, and `false` disables the notification.
+* `gcs_path`: The Cloud Storage bucket to upload Forseti violations as Cloud SCC findings.
+  [Sign-up for the Cloud SCC alpha program here!](https://services.google.com/fb/forms/commandcenteralpha/)
+
+## Usage
+
+You can invoke Forseti Notifier as part of the scheduled cron job's workflow
+or manually using the Forseti CLI.
+
+When the cron job runs the Notifier, it sends notifications on the
+latest violations from the previous scanner run.
+
+When you use the CLI, you can use flags to specify the violations that
+the notifier will run on.
+
+  * In the Foseti VM:
+  ```bash
+  $ forseti notifier --help
+  
+  # Send the violations from the last successful scanner run.
+  $ forseti notifier run
+
+  # Send the violations by inventory index id.
+  $ forseti notifier run --inventory_index_id <inventory index id>
+
+  # Send the violations by scanner index id.
+  $ forseti notifier run --scanner_index_id <scanner index id>
+  ```
+
+## What's next
+
+- Read more about
```

Comment:
Delete this bit and the trailing hard return (should be a single bullet starting with "Learn how...").

Reply:
Done